src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,12 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeeName, employees) => {
+    console.log(employeeName);
+    console.log(employees);
+    const empName = employees.map((nameList) => `${nameList.name}`);
+    const empRole = employees.map((nameList) => `${nameList.role}`);
+    console.log(empName);
+    console.log(empRole);
+    console.log(empRole[empName.length-1]);
+    return empRole[empName.length-1];
```

Comment:
So, this doesnt work. It passes the test, but will only ever return the role value for the last element in the array and since thats what youre testing for, it passes. Lets break this down a bit:

Youre trying to find a single person object in the provided array that matches the name given, then return that persons role. So the first thing to do is work out the right method to find them. There is a native js method called `find` which will do just that [docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)
We can say
```
return employees.find((employee) => employee.name === employeeName)
```
here we are looking through the array and trying to match the name property of each element in that array with the given `employeeName` value. If it finds one, it will return that object alone.
Now we need to get the value of `role` on this object, which we can do by appending `.role` onto the end of it:
```
  return employees.find((employee) => employee.name === employeeName).role;
```
Also, console logs are the Valerie of code reviews ;)